openwisp_controller/config/base/vpn.py

```diff
@@ -141,6 +149,14 @@ def get_context(self):
             c.update([('cert', self.cert.certificate), ('key', self.cert.private_key)])
         if self.dh:
             c.update([('dh', self.dh)])
+        # NOTE: Is this required?
+        if self.subnet:
+            c.update(
+                {
+                    'subnet': str(self.subnet.subnet),
+                    'subnet_prefix': str(self.subnet.subnet.prefixlen),
```

Comment:
maybe we can call it `subnet_prefixlen` for consistency, what do you think?

Reply:
Made this change! 👍🏼